tests/components/ozw/test_light.py

```diff
@@ -350,6 +350,48 @@ async def test_light(hass, light_data, light_msg, light_rgb_msg, sent_messages):
     assert state.attributes["color_temp"] == 153
 
 
+async def test_pure_rgb_dimmer_light(
+    hass, light_pure_rgb_dimmer_data, light_msg, light_pure_rgb_msg, sent_messages
+):
+    """Test light with no color channels command class."""
+    receive_message = await setup_ozw(hass, fixture=light_pure_rgb_dimmer_data)
+
+    # Test loaded
+    state = hass.states.get("light.kitchen_rgb_strip_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["supported_features"] == 17
+
+    # Test setting hs_color
+    new_color = [300, 70]
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.kitchen_rgb_strip_level", "hs_color": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 2
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 122257425}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)